Kernel/Graphics/DisplayConnector.cpp

```diff
@@ -188,22 +188,24 @@ ErrorOr<void> DisplayConnector::initialize_edid_for_generic_monitor(Optional<Arr
     return {};
 }
 
-void DisplayConnector::set_edid_bytes(Array<u8, 128> const& edid_bytes)
+void DisplayConnector::set_edid_bytes(Array<u8, 128> const& edid_bytes, bool might_be_invalid)
 {
     memcpy((u8*)m_edid_bytes, edid_bytes.data(), sizeof(m_edid_bytes));
```

Comment:
m_edid_bytes shouldn't need this cast right? `it's an unsigned char[128]`. Begs some other questions about why we're using a raw char array and a memcpy here, but that's not particularly relevant to this review :).